src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -196,12 +196,15 @@ public async Task RemoveAsync(IAccount account, CancellationToken cancellationTo
                 await UserTokenCacheInternal.RemoveAccountAsync(account, requestContext).ConfigureAwait(false);
             }
 
-            if (AppConfig.IsBrokerEnabled && ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth())
```

Comment:
We don't need this abstraction. Brokers can just do nothing if they don't support silent auth. Only the ios broker does not support this.